tests/test_main.py

```diff
@@ -4,122 +1,20 @@
+# tests/test_main.py
+import unittest
+from car_wash_station import CarWashStation  # Импортируем модуль из корня проекта
 
-from app.main import Car, CarWashStation
+class TestCarWashStation(unittest.TestCase):
+    def setUp(self):
+        self.station = CarWashStation()
 
+    def test_add_car(self):
+        self.station.add_car('Car 1')
+        self.assertIn('Car 1', self.station.cars)
 
-def test_car():
-    bmw = Car(2, 3, "BMW")
-    assert bmw.comfort_class == 2, "Class Car should store 'comfort_class'"
-    assert bmw.clean_mark == 3, "Class Car should store 'clean_mark'"
-    assert bmw.brand == "BMW", "Class Car should store 'brand'"
+    def test_remove_car(self):
+        self.station.add_car('Car 1')
+        self.station.remove_car('Car 1')
```

Comment:
The `remove_car` method is not defined in the `CarWashStation` class. You need to implement this method or adjust the test to match the available methods.